content/tutorials/portenta-h7/por-ard-flash/content.md

```diff
@@ -252,6 +255,19 @@ void loop()
 }
 ```
 
+
+
+## 6. Upload the Code
+
+Once you have uploaded the sketch to the board, open the Serial Monitor when the board starts it will display information about the flash memory.
```

Comment:
I guess there is something that is not working on my sketch, but I am not 100% sure about if it is because of my configuration or because of the program.
Looking at the program, I understood that we are saving on the Flash memory a "Hello World" sentence that would be printed once we reset the board but in my case, I always get the same information (the one in the screenshot)